src/http_microhttpd.c

```diff
@@ -451,7 +491,30 @@ static int preauthenticated(struct MHD_Connection *connection,
 		else
 			redirect_url = get_redirect_url(connection);
 
-		if (try_to_authenticate(connection, client, host, url)) {
+		if (config->bin_voucher) {
+			if(is_alphanum(client->token)) {
+				int ret = -1;
+				char *cmd = NULL, msg[255];
```

Comment:
It is easier to understand to put each variable declarations on a line of its own.